content/en/real_user_monitoring/frustration_signals/_index.md

```diff
@@ -0,0 +1,153 @@
+For access to frustration signals, contact <a href="/help">Datadog Support</a>.
+</div>
+
+Frustration signals identify the highest points of user friction in your application by surfacing moments where users exhibit frustration. 
+
+RUM collects three types of frustration signals:
+
+Rage Clicks
+: A user clicks on an element more than three times in a one-second rolling window.
+
+Dead Clicks
+: A user clicks on a static element thinking it is interactive.
+
+Error Clicks
+: A user clicks on an element right before a JavaScript error occurs.
+
+## Requirements
+
+First, you need the Browser RUM SDK version >= 4.9.0.
+
+To start collecting frustration signals, add the following to your SDK configuration:
+
+```
+DD_RUM.init({
+  trackInteractions: true,
+  trackFrustrations: true
+})
+```
+
+Frustration signals require actions. Enabling `trackFrustrations` automatically enables `trackInteractions`.
+
+## Usage
+
+Frustration signals appear as high-level data points representing sources of user frustration on the [**RUM Applications** page][1]. The [RUM Explorer][2] displays a count of frustration signals as a column in the **List** visualization. 
+
+### Application list
+
+Hover over the list of browser sessions and click on a session to observe a user's frustrated click behavior. Or, click **Frustrated Sessions** to access sessions with a frustration signal. 
+
+### Explore the frustration signals dashboard
+
+The **Frustration Signals** dashboard provides an overview of frustration levels across your application, showing you topics like the most frustrated users and pages with the highest amount of frustration signals.
+
+You can clone this dashboard and customize it to fit your needs. For more information, see [Frustration Signals Dashboard][3].
+
+### Search for frustration signals
+
+Search through all the data collected by RUM in the [RUM Explorer][4] to surface trends on frustration signals, analyze patterns with greater context, or export them into [dashboards][5] and [monitors][6].
+
+Enter a facet in the search query to start your search. Available search fields include:
+
+Frustration Type
+: Find actions with frustration signals. For example, if you want to see any actions that had a rage click, add `action.frustration_type:rage_click` to the search query.
+
+Frustration Count
+: Find sessions and views where any frustration signal occurred. For example, if you want to find any user sessions or views that had at least one frustration signal occur, add `session.frustration.count:>0` or `view.frustration.count:>0` to the search query.
+
+#### Sessions
+
+Click on a session with a value in the **Frustration Count** column to examine the user frustration detected. You can see the type of signal (ranging from `rage click`, `dead click`, or `error click`) and the event timeline that shows what occurred during the session.
+
+#### Views
+
+Click on a view to identify if a user was frustrated with the `frustration detected` tag.
```

Comment:
suggesting to reword "Click on a view to identify if a user was frustrated on a specific page with the 'frustration detected' tag